### DISCUSSION TITLE
Vamos adicionar badges no README?

### DISCUSSION THREAD

**Discussion by ortegavan** [techexpertspro/portal-ascepa#46] [Ideas]
Que tal deixarmos nosso README mais colorido? 
Minha sugestão é adicionar alguns badges do [Shields.io](https://shields.io) ao arquivo =)
Particularmente gosto do last commit, code style with prettier e tag. Tenho estas no [README do meu projeto da Mentoria](https://github.com/ortegavan/ecommerce).
Que mais podemos adicionar?  
<img width="924" alt="Captura de Tela 2024-06-10 às 19 03 18" src="https://github.com/techexpertspro/portal-ascepa/assets/30645960/0c5a5e60-5be1-4d05-8600-4b5da365d679">

**Comment by FelipeDuarteLuna** [comment]
Bom dia Ortegavan,

Concordo com você em utilizar o BADGES mas o principal é se a integração com a pipeline foi quebrado ou não, ao fazer o pull request.

![image](https://github.com/techexpertspro/portal-ascepa/assets/29357935/e58d0167-420a-465d-bb4b-e51c7fb167f8)

**Comment by andrewarosario** [comment]
Eu gosto muito também! Bora utilizar!
Outras badges que podemos adicionar no futuro quando tivermos essa integração: Sonar, Commitzen e Semantic Release
<img width="478" alt="image" src="https://github.com/techexpertspro/portal-ascepa/assets/21269337/108335f7-a8c4-46de-981f-1c2181b3cf61">

**Comment by ortegavan** [comment]
Shoooooow meus queridos! Vou esperar o Paolo finalizar a primeira versão, e aí na sequência já crio a task e o @Agemiro e eu fazemos!

**Reply by paoloalmeida** [reply]
Boooa @ortegavan!
Coloquei você e o @Agemiro já la na tarefa também, e hoje eu libero uma primeira versão... Daí vocês já complementam a ideia ai.
Acho total válido ter badges!

**Comment by andrewarosario** [comment]
Para melhor organização, vou transferir essa discussion para uma Issue e colocar o nome da @ortegavan e do @Agemiro, beleza? 😉

**Comment by andrewarosario** [comment]
Issue criada: #69
